modules/platformioBidAdapter.js

```diff
@@ -1,63 +1,125 @@
-var bidfactory = require('src/bidfactory.js');
-var bidmanager = require('src/bidmanager.js');
-var adloader = require('src/adloader.js');
-var utils = require('src/utils.js');
-var CONSTANTS = require('src/constants.json');
-var adaptermanager = require('src/adaptermanager');
 

```

Comment:
Preferable to start files with content rather than a blank line, line can be removed